src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2145,31 +2145,56 @@ private void VisitStatementsWithLocalFunctions(BoundBlock block)
 
         public override BoundNode? VisitLocalFunctionStatement(BoundLocalFunctionStatement localFunc)
```

Comment:
>localFunc [](start = 91, length = 9)

Not part of this change, but consider naming parameter `node`, to match other `Visit` methods and to remove the confusion between `localFunc` and `localFuncSymbol`.